string drone_lon
string drone_lat
string drone_height
string MGCP_lon
string MGCP_lat
string MGCP_height
string MGCP_groundSpeed
string MGCP_heading
